master/buildbot/worker/openstack.py

```diff
@@ -72,7 +73,7 @@ def __init__(self, name, password,
         self.client_version = client_version
         self.novaclient = client.Client(client_version, os_username,
                                         os_password, os_tenant_name,
-                                        os_auth_url)
+                                        os_auth_url, region_name=region)
```

Comment:
I would replace `region_name` with a dictionary (e.g. `**client_kwargs`) to cover even more use cases. The `Client` class has many parameters that cannot be used fron the OpenstackLatentWorker.